.goreleaser.yml

```
@@ -1,22 +1,23 @@
 builds:
-  - main: ./cmd/sqs/main.go
+  - main: ./cmd/connector/main.go
     goos:
       - darwin
       - linux
       - windows
     env:
       - CGO_ENABLED=0
     ldflags:
-      - "-s -w -X 'github.com/conduitio-labs/conduit-connector-amazon-sqs.version={{ .Tag }}'"
+      - "-s -w -X 'github.com/conduitio/conduit-connector-connectorname.version={{ .Tag }}'"
```

This was actually correct before
```suggestion
      - "-s -w -X 'github.com/conduitio-labs/conduit-connector-sqs.version={{ .Tag }}'"
```